packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -108,6 +109,11 @@ abstract class ProgressIndicator extends StatefulWidget {
   /// {@endtemplate}
   final String? semanticsValue;
 
+  /// The Progress Indicator's line endings
+  ///
+  /// Defaults to [StrokeCap.square]
+  final StrokeCap strokeCap;
```

Comment:
Is this for the value? Or the track? It may be beneficial to add both options where relevant, like the linear indicator.